configs/config.yaml

```diff
@@ -11,7 +11,7 @@ defaults:
   logging_steps: 1
   add_special_tokens: false
   use_fast_tokenizer: false
-  num_gpus: 4
+  num_gpus: 2
```

Comment:
Do I need to change `num_gpus` when I need other than 2 gpus?

Reply:
That's the problem I do not understand how to solve. `num_gpus` is needed only to calculate a number of accumulation steps per GPU, since in config we set total batch accum and minibatch size per device.

Reply:
I fixed this issue in [2effd4b] commit. Removed n_gpus and n_nodes from config. Now it takes this information from accelerate.